app/_locales/en/messages.json

```diff
@@ -6815,5 +6815,11 @@
   },
   "zeroGasPriceOnSpeedUpError": {
     "message": "Zero gas price on speed up"
+  },
+  "smartTransactionsEnabledMessage": {
+    "message": "You're now protected with MetaMask Smart Transactions"
+  },
+  "smartTransactionsLearnMore": {
```

Comment:
This one is not needed, we already have the "learnMore" one.

Reply:
Good call, look at all of these:
![image](https://github.com/user-attachments/assets/7318d24a-03ea-4378-b6a5-1a74f7d953b4)